src/read.c

```diff
@@ -3701,6 +3797,56 @@ static avifResult avifDecoderDataFindAlphaItem(avifDecoderData * data,
     return AVIF_RESULT_OK;
 }
 
+#if defined(AVIF_ENABLE_EXPERIMENTAL_GAIN_MAP)
+// Finds a 'tmap' (tone mapped image item) box associated with the given 'colorItem'.
+// If found, fills 'toneMappedImageItem' and  sets 'gainMapItemID' to the id of the gain map
+// item associated with the box. Otherwise, sets 'toneMappedImageItem' to NULL.
+// Returns AVIF_RESULT_OK on success (whether or not a tmap box was found).
```

Comment:
Will a grid of `tmap` cells be forbidden in the specification? Otherwise, there should be a comment or TODO mentioning the lack of support for that pattern at decoding. It is fine not to support that at encoding, as discussed.

Reply:
Not sure if it will be allowed. Added a comment/TODO.